js/hang-ui/src/Components/publish/CameraSourceButton.tsx

```diff
@@ -5,23 +5,17 @@ import { PublishUIContext } from "./PublishUIContextProvider";
 export default function CameraSourceButton() {
 	const context = useContext(PublishUIContext);
 	const onClick = () => {
-		const hangPublishEl = context?.hangPublish();
-		if (!hangPublishEl) return;
-
-		if (hangPublishEl.source.peek() === "camera") {
+		if (context?.hangPublish.source.peek() === "camera") {
```

Comment:
Can we avoid `context` being `undefined`? Straight from the docs: https://docs.solidjs.com/reference/component-apis/use-context#recommended-usage

Reply:
Yeah, I just don't like the magic of global variables like `useContext` but whatever works 👍